src/pages/index.astro

```diff
@@ -102,27 +102,6 @@ import { PAGES } from "../utils/constants.js";
       </div>
     </div>
   </section>
-  <ContentMedia imgSrc="/accessible-components.webp">
```

Comment:
I recommend removing the change you made in this file since you already did that in https://github.com/CareerCatalyst/Career-Catalyst/pull/63/files. That way we can keep the work separate.

Reply:
Resolved in commit 29eee7d